package.json

```
@@ -65,6 +65,7 @@
     "fuse.js": "7.0.0",
     "lodash.isequal": "4.5.0",
     "lodash.mergewith": "4.6.2",
+    "loglevel": "^1.9.2",
```

We usually pin our deps to specific versions

```suggestion
    "loglevel": "1.9.2",
```

---

Done.